### ISSUE TITLE
Fix composite PK uniqueness check

### ISSUE THREAD

**Issue by rgieseke** [frictionlessdata/goodtables-py#215]
Is it possible to check for unique-ness over two columns, e.g. for a composite primary index?
Even if I set both columns unique-ness as false, I get validation errors with 1.4.1

```json
{
    "name": "testing",
    "resources": [
        {
            "name": "data",
            "path": "data.csv",
            "schema": {
                "fields": [
                    {
                        "name": "name",
                        "type": "string",
                        "constraint": {
                            "unique": false
                        }
                    },
                    {
                        "name": "year",
                        "type": "year",
                        "constraint": {
                            "unique": false
                        }
                    },
                    {
                        "name": "amount",
                        "type": "number"
                    }
                ],
                "primaryKey": ["name", "year"]
            }
        }
    ]
}
```

```csv
name,year,amount
Taxes,1990,10000.5
Taxes,1991,10001
Fees,1990,2000.5
Fees,1991,2001
```

I can disable the unique check, but was curious if it's possible - otherwise `goodtables` has already been super helpful in finding tiny errors!

**Comment by roll** [comment]
@rgieseke 
I'm considering current behavior as a bug. There was intention to treat primaryKey as unique field but current implementation doesn't support composite primary keys. So e.g. this descriptor should work:

```{
    "name": "testing",
    "resources": [
        {
            "name": "data",
            "path": "data.csv",
            "schema": {
                "fields": [
                    {
                        "name": "name",
                        "type": "string",
                    },
                    {
                        "name": "year",
                        "type": "year",
                    },
                    {
                        "name": "amount",
                        "type": "number"
                    }
                ],
                "primaryKey": ["name", "year"]
            }
        }
    ]
}
```

Looking forward to fix it on this or next week.

**Comment by rgieseke** [comment]
Cool, just saw this was introduced in 1.4.

**Comment by roll** [comment]
Fixed in `goodtables@1.4.2`
